src/mds/Server.cc

```diff
@@ -4835,6 +4835,7 @@ void Server::handle_client_readdir(MDRequestRef& mdr)
       if (logger)
           logger->inc(l_mdss_cap_acquisition_throttle);
 
+      mdr->mark_event("cap_acquisition_throttle");
```

Comment:
Should we do the same in `handle_client_readdir_snapdiff()` too ?

Reply:
Good catch, we surely do! My bad, I found that instance at the beginning but then totally forgot about it.

Reply:
Now I just need to figure out how to hit that flow in test. Any pointers?

Reply:
There is a way to trigger that from ceph-fuse mount, but src/test/libcephfs/snapdiff.cc might be a tiny bit easier too.

Reply:
OK now that flow is also tested